media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java

```diff
@@ -60,6 +67,16 @@
 @Model(adaptables = SlingHttpServletRequest.class)
 public class ResourceMedia {
 
+  /**
+   * Name prefix for request attributes that will be put into the media builder properties
+   */
+  private static final String REQUEST_ATTRIBUTE_PREFIX = "mediaProp:";
```

Comment:
i would prefer using the same name for the parameter as in the MediaBuilder interface - meaning "property" instead of "mediaProp"